python/aswfdocker/tests/test_utils.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import logging
+import tempfile
+import os
+
+from click.testing import CliRunner
+
+from aswfdocker import utils
+from aswfdocker.cli import aswfdocker
+
+
+class TestUtils(unittest.TestCase):
+    def test_get_docker_org(self):
+        self.assertEqual(utils.get_docker_org("", ""), "aswftesting")
+        self.assertEqual(
+            utils.get_docker_org(
+                "https://github.com/AcademySoftwareFoundation/aswf-docker",
```

Comment:
Since we're talking about potentially transitioning to a shorter GitHub organization name, perhaps there should be a centralized config for GITHUB_ASWF_ORG_URL or something like that.